packages/esm-patient-list-app/src/api/api-remote.ts

```diff
@@ -136,6 +146,17 @@ export async function addPatientToList(data: AddPatientData, ac = new AbortContr
   return postData(`${cohortUrl}/cohortmember`, data, ac);
 }
 
+export async function removePatientFromList(cohortMembershipUuid: string) {
+  const ac = new AbortController();
+  return postData(
+    `${cohortUrl}/cohortmember/${cohortMembershipUuid}`,
+    {
+      endDate: new Date(),
```

Comment:
@dkayiwa what should happen after removing a patient from a list using this function? Should subsequently fetching the list return a response whose `endDate` property is set to the value passed above? And further, should the frontend filter the returned cohort data to remove entries with an `endDate` property?

Reply:
After removing a patient from the list, the original REST call which was used, should return a list which does not include the removed patient.